verification_token/models.py

```diff
@@ -59,13 +59,14 @@ def exists_valid(self, obj, key, slug=None):
                 return True
         return False
 
-    def filter_active_tokens(self, obj, slug=None, key=None):
+    def filter_active_tokens(self, obj_or_class, slug=None, key=None):
         qs = self.filter(
             is_active=True,
             slug=slug,
-            content_type=ContentType.objects.get_for_model(obj.__class__),
-            object_id=obj.pk
+            content_type=ContentType.objects.get_for_model(obj_or_class),
         )
+        if inspect.isclass(obj_or_class):
```

Comment:
Tohle podle mě vrátí True když to bude jen classa např. `Customer`. Není lepší čekovat jestli to má atribut `pk`?

Reply:
Škoda že knihovna nemá testy, hned by bylo jasno.

Reply:
Udělej pls testy a bude hned jasno.

Reply:
the same for checking attr.
```
In [4]: hasattr(Customer, 'pk')                                                               
Out[4]: True
```
@matllubos should it actually be `isinstance(obj_or_class, django.db.Model)` instead so just instances of Model are filtered?

Reply:
Yes isinstance(obj_or_class, django.db.Model) is the right solution

Reply:
And please write test

Reply:
@matllubos @rubickcz ok, it's fixed now and tests were added